src/content/app/tools/blast/views/blast-form/BlastFormContextContainer.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactNode, useState } from 'react';
+
+import useBlastForm from 'src/content/app/tools/blast/hooks/useBlastForm';
+
+type SequenceValidity = {
+  updateSequenceValidity: (index: number | null, status: boolean) => void;
+  removeSequenceValidity: (index: number) => void;
+  sequencesValidity: boolean[];
+};
+
+export const BlastFormContext = React.createContext<
+  SequenceValidity | undefined
+>(undefined);
```

Comment:
Is the `undefined` argument required? I thought if you don't pass any arguments, it will be undefined.

Reply:
If i removed the `undefined` in the argument, the function complains saying it needs at least one argument passed. This is the error:
![image](https://user-images.githubusercontent.com/6040010/175886423-bed0510f-c1b1-45a8-92d7-cc7d09bb2eb6.png)

Reply:
Ok, I thought the argument for `createContext` was optional; but apparently not, according both to the docs and type definitions.